CHANGELOG.md

```diff
@@ -49,6 +49,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Modified Traffic Router to separate availability statuses between IPv4 and IPv6.
 - Modified Traffic Portal and Traffic Ops to accept IPv6 only servers.
 - Updated Traffic Monitor to default to polling both IPv4 and IPv6.
+- Traffic Vault - Enable TLSv1.1
+- Compiling Traffic Ops, Traffic Monitor, or Traffic Stats, or Grove now requires Go version 1.14.
```

Comment:
Could we say "x, y, and z are now compiled with Go version 1.14" instead? It's hard to say that the components truly require 1.14 until they are all actually using/dependent upon something in 1.14 only.

Reply:
Reworded in e93ab9c621.

Reply:
2b159450fb *does* bump the minimum version needed to compile Traffic Ops to 1.12, due to using `tls.VersionTLS13`. Is that worth mentioning?

Reply:
Thanks, just trying to keep the changelog informative since I'll most likely be using it for the 4.1 release notes. I don't think minimum version is worth mentioning. This LGTM now